### PR TITLE
fix: globalThis with CSP compatibility

### PR DESCRIPTION
**Pull request by himself65**
Fixes: https://github.com/istanbuljs/istanbuljs/issues/621

### CODE REVIEW

File: packages/istanbul-lib-instrument/test/util/verifier.js [hunk 1]
```diff
@@ -26,7 +26,10 @@ function annotatedCode(code) {
 }
 
 function getGlobalObject() {
-    return new Function('return this')();
+  return typeof globalThis === 'object' ? globalThis :
```
Comment (bcoe) [inline packages/istanbul-lib-instrument/test/util/verifier.js:29]: The nested ternary is a bit confusing, mind breaking this on to a couple lines?

**Comment by himself65** [comment]
Fixed

**Comment by himself65** [comment]
any update?

**Review by SimenB — APPROVED** [review]
👍

**Comment by SimenB** [comment]
https://github.com/istanbuljs/istanbuljs/releases/tag/istanbul-lib-instrument-v6.0.1

**Pull request merged**